core/src/Streamly/Internal/FileSystem/Path/Common.hs

```diff
@@ -426,62 +414,6 @@ isAbsolute Windows arr =
 -- leave this to the programmer. In typed paths we can allow "./x" in segments.
 -- XXX Empty path can be taken to mean "." except in case of UNC paths
 
--- | Any path that starts with a separator, @./@ or a drive prefix is a rooted
```

Comment:
Why did you remove these docs?
The doctests here run on the CI.

Reply:
They are in the PosixPath and WindowsPath modules now, for user documentation. We do not want to maintain them at two places.